com.unity.cinemachine/Documentation~/InputSystemComponents.md

```diff
@@ -0,0 +1,71 @@
+# Input System Components
```

Comment:
This page is not linked to anywhere.  You should add an entry in the table of contents.
Also, you should change the title to "Using Input System Components with Cinemachine".

Reply:
Jess is looking at it. Gonna wait for her changes but yes will do.